opendevin/core/utils/readers.py

```diff
@@ -0,0 +1,538 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+INSTALL
+-------
+pip install openai --upgrade
+pip install python-docx
+pip install markdown
+pip install PyPDF2
+pip install openpyxl
+pip install beautifulsoup4
+pip install pylatexenc
+pip install python-pptx
+pip install xlrd
+pip install base64
+pip install opencv-python
+"""
```

Comment:
Maybe we should move those to our dependency file.

Reply:
`pyproject.toml`, specifically